omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-SummaryPage.xml

```diff
@@ -1090,22 +1090,17 @@
 
                             <span class="ARTCareEntryPointEnableDisable1">
                                         <obs conceptId="90200"
-                                             answerConceptIds="90012,90016,99593,90019,90013,90015,90018,90002"
-                                             answerLabels="eMTCT,TB,YCC,Outreach,Out Patient,STI,Inpatient,Other"
+                                             answerConceptIds="90012,90016,99593,99610,90019,90013,90015,90018,90002"
```

Comment:
@slubwama Correct the naming of the concept 99610 from "SMS" to "SMC"

Reply:
@slubwama use 99620 as the care entry point instead of 99610 which is part of the coded values